x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx

```diff
@@ -186,25 +186,31 @@ export const BenchmarksSection = ({
             />
           </EuiFlexItem>
           <EuiFlexItem grow={dashboardColumnsGrow.third}>
-            <RisksTable
-              compact
-              data={cluster.groupedFindingsEvaluation}
-              maxItems={3}
-              onCellClick={(resourceTypeName) =>
-                navToFailedFindingsByClusterAndSection(cluster, resourceTypeName)
-              }
-              viewAllButtonTitle={i18n.translate(
-                'xpack.csp.dashboard.risksTable.clusterCardViewAllButtonTitle',
-                {
-                  defaultMessage: 'View all failed findings for this {postureAsset}',
-                  values: {
-                    postureAsset:
-                      dashboardType === CSPM_POLICY_TEMPLATE ? 'cloud account' : 'cluster',
-                  },
+            <div
+              style={{
+                paddingRight: euiTheme.size.base,
+              }}
+            >
```

Comment:
this is the only change in this file, it makes the right side padding of the benchmark's risks table aligned with the padding from the summary section